### ISSUE TITLE
gnng: Display Interfaces Without IP Addresses

### ISSUE THREAD

**Issue by supertylerc** [trigger/trigger#273]
`gnng` appears to only list interfaces with IP addresses.  This is slightly confusing, as the documentation states:

> gnng - Fetches network devices interfaces and displays them in a table view.
> 
> Fetches interface information from routing and firewall devices. This includes
> network and IP information along with the inbound and outbound filters that
> may be applied to the interface. Works on Juniper, Netscreen, Foundry, and Cisco
> devices.

It _includes_ network and IP information, but the documentation does not state that it _excludes_ interfaces without IP addresses.  In many cases, it may be desirable to see all ports, regardless of whether it's a routed interface or not.

Even if it's undesirable to list ports that don't have IP addresses, maybe the documentation should be updated at the least to indicate it won't retrieve interfaces without IPs.

**Comment by jathanism** [comment]
There are two flags that attempt to modify this behavior, but I seem to recall there being some more flaws to them working as intended:

```
  -d, --include-disabled
                        include disabled interfaces.
  -u, --include-unnumbered
                        include un-numbered interfaces.```

**Comment by jathanism** [comment]
Yeah this came up in https://github.com/trigger/trigger/issues/257, item 4.

**Comment by jathanism** [comment]
However, on a device I have that has an unnumbered interface, `-u/--include-unnumbered` does seem to work as expected:

```
$ gnng juniper-fw1 --include-unnumbered
DEVICE: juniper-fw1
Interface  | Addresses | Subnets | ACLs IN | ACLs OUT | Description
-------------------------------------------------------------------
ge-0/0/0.0 |           |         |         |          |
```

For comparison without `--include-unnumbered`:

```
$ gnng juniper-fw1
DEVICE: juniper-fw1
Interface | Addresses | Subnets | ACLs IN | ACLs OUT | Description
---------------------
```

**Comment by supertylerc** [comment]
Is "unnumbered" to mean interfaces that might be L2 ports, or is that unnumbered as in "ip unnumbered"?

**Comment by jathanism** [comment]
@supertylerc "ip unnumbered" although I don't think the distinction is really that well-defined. It's pretty much "any interface without an IP address".
